src/Resources/config/services.xml

```diff
@@ -27,6 +27,7 @@
                     </argument>
                 </service>
             </argument>
+            <argument />
```

Comment:
Add a comment here: `<!-- strict mode -->` (but also, I think we may move this config to another service)